Source/DafnyDriver/BoogieXmlConvertor.cs

```diff
@@ -24,33 +24,25 @@ namespace Microsoft.Dafny {
   /// </summary>
   public static class BoogieXmlConvertor {
 
-    public static void RaiseTestLoggerEvents(string fileName, string loggerConfig) {
-      string loggerName;
-      Dictionary<string, string> parameters;
-      int semiColonIndex = loggerConfig.IndexOf(";");
-      if (semiColonIndex >= 0) {
-        loggerName = loggerConfig[..semiColonIndex];
-        var parametersList = loggerConfig[(semiColonIndex + 1)..];
-        parameters = parametersList.Split(",").Select(s => {
-          var equalsIndex = s.IndexOf("=");
-          return (s[..equalsIndex], s[(equalsIndex + 1)..]);
-        }).ToDictionary(p => p.Item1, p => p.Item2);
-      } else {
-        loggerName = loggerConfig;
-        parameters = new();
-      }
-
```

Comment:
It makes sense to parse all logger configs like this as the syntax is not logger-specific.

Reply:
Yep, that's back in now, and makes logging to a file possible again.